reference/checkouts.v3.yml

```diff
@@ -1202,35 +1145,12 @@ definitions:
                               type: number
                               format: double
                             discounted_amount:
-                              description: >-
-                                The discounted amount applied within a given
-                                context.
+                              description: The discounted amount applied within a given context.
                               type: number
                               format: double
                       coupons:
                         description: ''
-                        type: array
-                        items:
-                          title: Applied Coupon
-                          type: object
-                          properties:
-                            code:
-                              description: the coupon code
-                              type: string
-                            id:
-                              description: The coupon ID.
-                              type: string
-                            coupon_type:
-                              description: Key name to identify the type of coupon.
-                              type: string
-                            discounted_amount:
-                              description: >-
-                                The discounted amount applied within a given
-                                context.
-                              type: number
-                              format: double
-                          required:
-                            - code
+                        type: number
```

Comment:
I believe this should be `type: object`. The AppliedCoupon model has all of the applicable properties: id, code, discounted_amount, coupon_type.